include/UartSerial.h

```diff
@@ -76,6 +79,11 @@ class UartSerial : public Stream {
     struct termios _tio_original_config;
     struct termios _tio_config;
     serialEvent _bytes_available_callback;
+    void *_callbackContext;
```

Comment:
I have since decided to shut up about variable names, you'll find an olive branch below. :smiley: